flipperdevices/flipperzero-firmware
# [FL-1338] Initialize USB as soon as possible, fix usb stack lockup

# What's new

- Move USB initialization to the beginning, fix stack lockup issues on some host controllers (Intel Wildcat)

# Verification 

- Compile and upload
- Connect device to PC, reset device.
- Check kernel logs: VCP should reappear after reset.

# Checklist (do not modify)
